cluster.go

```diff
@@ -2,6 +2,7 @@ package gossipcache
 
 import (
```

Comment:
you have a whitespace issue on https://github.com/darvaza-proxy/gossipcache/blob/35357924f5f69c94d5a08c21c811343ee874b6f6/cluster.go#L293